cape_privacy/pandas/transformations/tokenizer.py

```diff
@@ -2,6 +2,7 @@
 import secrets
 
 import pandas as pd
+from Crypto.Cipher import AES
```

Comment:
who made this library and why is it capitalized ☹️

Reply:
yeah, try to look away :)